test/test_scatter_gather_ops.py

```diff
@@ -264,6 +264,33 @@ def test_scatter_reduce_amin(self, device, dtype):
                     expected_result[2] = 0
                 self.assertEqual(input, expected_result)
 
+    @onlyCPU
+    @dtypes(*get_all_dtypes(include_half=True, include_bfloat16=True, include_complex=False))
+    def test_scatter_expanded_index(self, device, dtype):
+        def helper(dim_size, idx_size, slice_size):
+            input = torch.randn(dim_size, slice_size)
```

Comment:
Even though 2D is a typical case for GNN, suggest to add cases on >2D too since your fast path is not specialized for 2D.

Reply:
Good point, add 3d and 4d scenarios!